samples/kvsWebRTCClientMasterGstSample.c

```diff
@@ -410,7 +326,10 @@ INT32 main(INT32 argc, CHAR* argv[])
 
     pSampleConfiguration->videoSource = sendGstreamerAudioVideo;
     pSampleConfiguration->mediaType = SAMPLE_STREAMING_VIDEO_ONLY;
+
+#ifdef ENABLE_GST_SAMPLE_RECEIVER
```

Comment:
I dont think I understand the requirement for this cmake flag in the sample. Also, let us avoid creating new CMake based flags. We always had this receive callback irrespective of what the pipeline looked like. I think this should be a default. If we do not want to do that, it should be a flag in pSampleConfiguration that should be set.

Reply:
As per our offline discussion, I've created a new sample application and removed the cmake flag